cli/pom.xml

```diff
@@ -62,6 +55,12 @@
       <version>${mina-sshd.version}</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcprov-jdk18on</artifactId>
+      <version>1.81</version>
+      <optional>true</optional>
+    </dependency>
```

Comment:
can this be removed in the future if https://github.com/apache/mina-sshd/issues/585 is implemented (so we use native java) without breaking anything?

If this adds other key formats/exchanges that where not present should they be documented as possibly to be removed in a future version so that the CLI could actually become FIPS compliant?